soh/soh/Enhancements/controls/InputViewer.cpp

```diff
@@ -380,47 +459,142 @@ void InputViewerSettingsWindow::DrawElement() {
         UIWidgets::PaddedSeparator(true, true);
 
         if (ImGui::CollapsingHeader("Buttons")) {
+
+            // gInputViewer.ButtonOutlineMode
+            UIWidgets::PaddedText("Button Outlines/Backgrounds", true, false);
+            UIWidgets::EnhancementCombobox(
+                "gInputViewer.ButtonOutlineMode", buttonOutlineOptions, BUTTON_OUTLINE_NOT_PRESSED,
+                !CVarGetInteger("gInputViewer.GlobalButtonOutlineMode", 1), "",
+                CVarGetInteger("gInputViewer.ButtonOutlineMode", BUTTON_OUTLINE_NOT_PRESSED));
+            UIWidgets::Tooltip(
+                "Sets the desired visibility behavior for the button outline/background layers. Useful for "
+                "custom input viewers.");
+
+            // gInputViewer.GlobalButtonOutlineMode
+            UIWidgets::EnhancementCheckbox("Use for all buttons", "gInputViewer.GlobalButtonOutlineMode", false, "",
```

Comment:
Small nitpick but i think `gInputViewer.GlobalButtonOutlineMode` is a bit misleading of a name. I think matching your use of useGlobalOutlineMode would make the most sense (i.e. `gInputViewer.UseGlobalOutlineMode`)

Reply:
That's a good point. It's been renamed to `UseGlobalButtonOutlineMode`